ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/JdbcLedgerDao.scala

```diff
@@ -348,7 +370,8 @@ private class JdbcLedgerDao(
       offset: Long,
       tx: Transaction,
       localImplicitDisclosure: Relation[EventId, Party],
-      globalImplicitDisclosure: Relation[AbsoluteContractId, Party])(
+      globalImplicitDisclosure: Relation[AbsoluteContractId, Party],
+      referencedContracts: List[(Value.AbsoluteContractId, AbsoluteContractInst)])(
```

Comment:
Hi Robert - might we worth renaming referencedContracts to divulgedContracts for consistency with the Update event which Jussi recent tweaked.